### DISCUSSION TITLE
无法登录的解决方案，如提示「请确保你的网络畅通」

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2425] [Developer] [answered]
> 备注：此问题已经解决。

**Comment by blankwz** [comment]
大佬，怎么登陆到 https://simpread.pro/developer ，UID的License怎么获取啊~     是否会自动登陆，我按要求 授权了 坚果云|Dropbox  ，一直以为是会自动登陆

**Reply by Kenshin** [reply]
License 请看这里 https://simpread.pro/api/#/如何开通?id=如何找到-license
UID  请看这里 https://simpread.pro/api/#/如何开通?id=如何找到-uid

这两个链接在首页就能访问到，如下图

![image](https://user-images.githubusercontent.com/81074/126885014-3aa9e191-f261-49d5-ba58-15e504663f38.png)

**Comment by cyberovid** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/36943015/130200485-30956487-a022-4ca2-852c-9f6be37d4f02.png)
![image](https://user-images.githubusercontent.com/36943015/130200540-5e1e4ae3-afa1-4f51-948d-f7672083ab01.png)
请问我uid和license都应该找对了，但是
![image](https://user-images.githubusercontent.com/36943015/130200749-3256ab93-a60e-4ed3-9a22-50e0c4a93b90.png)
没有填错，直接粘贴的 ，也没有填反。

**Reply by cyberovid** [reply]
我只是想给telebot来个token

**Reply by cyberovid** [reply]
坚果云同步，配置文件也在
![image](https://user-images.githubusercontent.com/36943015/130201314-fa4b031a-02a7-49f7-ac5d-cb1bfdf9809c.png)

**Reply by Kenshin** [reply]
如果 UID / License 没有填错的话，估计是 [这个问题造成的](https://simpread.pro/api/#/%E5%A6%82%E4%BD%95%E5%BC%80%E9%80%9A?id=%e6%8e%88%e6%9d%83%e5%9d%9a%e6%9e%9c%e4%ba%91%e7%bb%99%e7%ae%80%e6%82%a6)

**Reply by aka-phrankie** [reply]
开的都开了，我也确保我没填错。坚果云授权也对，可以同步。网络也开全局了。

**Reply by Kenshin** [reply]
@LIL-YAML 

「一直不行」是指什么呢？如果希望可以让我帮到你，可以详细说明或截图。

PS：Telegram bot 每天要处理 5K+ 的 URLs。

**Reply by YB-Kang** [reply]
作者您好，我也是同样卡在网络问题，请教下有没有什么办法解决？

**Reply by Kenshin** [reply]
@KYBIN-9 

嗨，请保证你的网络可以「正常」访问，细节请看这里 https://github.com/Kenshin/simpread/discussions/2425

**Reply by YB-Kang** [reply]
@Kenshin 我看过几遍了，也反复试了几次，都没问题啊，跟 @LIL-YAML 一样，开全局，坚果云授权，没填错，自动同步了，还是登陆不了

**Reply by Kenshin** [reply]
@KYBIN-9 

emmm... 把你的 UID / License 发到我邮箱里面（在我的 Github 上面能找到邮箱地址）

另外，提示什么？请截图看下。

**Reply by YB-Kang** [reply]
@Kenshin 发过去了

**Reply by YB-Kang** [reply]
@Kenshin @LIL-YAML 我刚才针对网络问题尝试和排查可能解决的方法，发现确实是全局的问题，已经搞定了。

**Reply by Kenshin** [reply]
@KYBIN-9 

搞定就好，为了某些原因，我把你的一段内容删除了，请理解。

**Reply by YB-Kang** [reply]
@Kenshin 好的，我记住了。感谢作者的辛勤付出。

**Reply by Kenshin** [reply]
@KYBIN-9 

嗨，忘记说了。
如果继续折腾 tg bot 可以看这个教程 https://github.com/Kenshin/simpread/discussions/2792

**Reply by YB-Kang** [reply]
@Kenshin 嗯。我正在折腾 开放平台 这块
